jupyterhub/user.py

```diff
@@ -905,6 +905,7 @@ async def spawn(self, server_name='', options=None, handler=None):
             # wait for spawner.start to return
             # run optional preparation work to bootstrap the notebook
             await maybe_future(spawner.run_pre_spawn_hook())
+            await spawner.apply_group_overrides()
```

Comment:
What's your thinking for running this after `pre_spawn_hook` instead of before?
https://github.com/jupyterhub/jupyterhub/blob/fb85cfb118af55c34a24ba20ff1bcb595325c5ab/jupyterhub/spawner.py#L909-L913

Reply:
Just missed it, fixed now.